### PR TITLE
implement backup delete

### PR DESCRIPTION
**Pull request by svt-kmcdaniel**
Signed-off-by: Ken McDaniel <kenneth.mcdaniel@hpe.com>

### CODE REVIEW

File: simplivity/resources/backups.py [hunk 1]
```diff
@@ -26,6 +26,18 @@ class Backups(ResourceBase):
     def __init__(self, connection):
         super(Backups, self).__init__(connection)
 
+    def delete_multiple_backups(self, backups, timeout=-1):
```
Comment (sijeesh) [inline simplivity/resources/backups.py]: Let's keep this method after the get_all method.

**Review by sijeesh — COMMENTED** [review]
Looks good, please update the endpoint support file.
https://github.com/HewlettPackard/simplivity-python/blob/master/endpoints-support.md

File: simplivity/resources/backups.py (hunk 1 above)
Comment (svt-kmcdaniel) [reply simplivity/resources/backups.py]: I moved the code block below def get_all as requested

**Review by sijeesh — APPROVED** [review]
LGTM

**Pull request merged**